.gitlab/spack_packages/sundials/package.py

```diff
@@ -788,23 +844,30 @@ def initconfig_package_entries(self):
             entries.append(cmake_cache_path("RAJA_DIR", spec["raja"].prefix))
             if "camp" in spec:
                 entries.append(cmake_cache_path("camp_DIR", spec["camp"].prefix.lib.cmake + '/camp'))
+            if "+rocm" in spec:
+                entries.append(cmake_cache_string("SUNDIALS_RAJA_BACKENDS", "HIP"))
 
         # Building with SuperLU_DIST
         if "+superlu-dist" in spec:
-            if spec.satisfies("@6.4.0:"):
+            #if spec.satisfies("@6.4.0:"):
+            if False:
```

Comment:
Can this be reverted to `if spec.satisfies("@6.4.0:"):`

Reply:
No, we need to always use the other way for now due to the issue with the RPATH in superlu-dist. I think leaving this block in here with `if False:` (or commented out) is fine, but we should add a comment on why we are doing so.